test/e2e-test-application/src/app/project/project.component.html

```diff
@@ -359,6 +370,24 @@ <h3 class="fd-layout-panel__title">Navigate</h3>
             >
             </app-code-snippet>
           </li>
+          <li class="fd-list__item">
+            <a
+              href="javascript:void(0)"
+              class="fd-link"
+              (click)="
+                linkManager()
+                  .fromClosestContext()
+                  .withParams({ feature: 'toggle' })
+                  .navigate('settings_ft?ft=test')
```

Comment:
I got an error when clicking on this node:

![Screenshot 2020-08-20 at 08 21 43](https://user-images.githubusercontent.com/18335489/90724076-363f4080-e2be-11ea-97dc-892b419a4daf.png)